sync/firewall/firewall_test.go

```diff
@@ -354,7 +354,7 @@ func TestNetworkFlagsTestnet(t *testing.T) {
 
 func TestNetworkFlagsLocalnet(t *testing.T) {
 	td := setup(t, nil)
-	td.state.TestParams.BlockVersion = 0x3f // changing genesis hash
+	td.state.Genesis().Params().BlockVersion = 0x3f // changing genesis hash
```

Comment:
Why Genesis here?

Reply:
changing test params won't change the version and test will fail.